ElectronicObserver/Window/Tools/ExpeditionRecordViewer/ExpeditionRecordViewModel.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using ElectronicObserver.Database.Expedition;
+using ElectronicObserver.KancolleApi.Types.ApiReqMission.Result;
+using ElectronicObserver.Window.Tools.SortieRecordViewer;
+using ElectronicObserverTypes;
+using System.Text.Json;
+using System.Collections.Generic;
+using ElectronicObserver.Data;
+
+namespace ElectronicObserver.Window.Tools.ExpeditionRecordViewer;
+
+public class ExpeditionRecordViewModel
+{
+	public ExpeditionRecord Model { get; }
+	public int? Id => Model?.Id;
+	public DateTime ExpeditionStart { get; }
+	public IFleetData? Fleet { get; }
+	public List<int>? ItemList { get; }
+	public object? MaterialList { get; }
+	public int MaterialFuel { get; }
+	public int MaterialAmmo { get; }
+	public int MaterialSteel { get; }
+	public int MaterialBaux { get; }
+	public int? ItemOneID { get; }
```

Comment:
Can you group these properties in some logical way? Either put item1/2 data together or group them by use case, currently they're just thrown around randomly.
![image](https://user-images.githubusercontent.com/40002167/233775649-4f5e6abf-2fac-4a1c-91ff-c3d4e7e6fb77.png)